features/org.wso2.carbon.email.mgt.server.feature/resources/email-admin-config.xml

```diff
@@ -17195,4 +17195,469 @@
+                                                                </tr>
+                                                            </tbody>
+                                                        </table>
+                                                    </td>
+                                                </tr>
+                                                <tr>
+                                                    <td style="-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;mso-table-lspace: 0pt;mso-table-rspace: 0pt;background-color: {{organization.color.background}};width: 100%;"
+                                                        width="100%" valign="top" align="left">
+                                                        <!-- BEGIN BODY // -->
+                                                        <table id="templateBody"
+                                                            style="width: 100%;max-width: 650px;-ms-text-size-adjust: 100%;-webkit-text-size-adjust: 100%;background-color: {{organization.color.background}};mso-table-lspace: 0pt;mso-table-rspace: 0pt;margin: auto;"
+                                                            width="100%" cellspacing="0" cellpadding="0" border="0">
+                                                            <tbody>
+                                                                <tr>
+                                                                    <td class="footerContent"
+                                                                        style="-webkit-text-size-adjust: 100%;-ms-text-size-adjust: 100%;mso-table-lspace: 0pt;mso-table-rspace: 0pt;font-family: '{{organization.font}}', Roboto, Verdana, Helvetica, sans-serif;font-size: 10px;line-height: 15px;text-align: left;padding: 10px 40px 25px;"
+                                                                        valign="top" align="left">
+                                                                        <p style="font-size: 14px; line-height: 140%; padding-top: 8px;">
+                                                                            <span
+                                                                                style="font-size: 12px; line-height: 16.8px; color: #808080;">You
+                                                                                received this email because you have an account in the
+                                                                                organization
+                                                                                <b>{{organization-name}}</b>. If you encounter any issues,
+                                                                                you may contact us at
+                                                                                <a
+                                                                                    class="wso2_orange3" href="mailto:{{organization.support.mail}}"
```

Comment:
organization.support.mail -> what's the placeholder value for this?

Reply:
This placeholder is an existing one. Seems like for the other places also this is not properly populated.

<img width="543" alt="image" src="https://github.com/wso2-extensions/identity-event-handler-notification/assets/19262008/aec306ae-e553-418e-a11e-e529bf279c64">

Reply:
This placeholder is available to be configured with branding